mythtv/libs/libmythtv/recorders/audioinputalsa.cpp

```diff
@@ -271,9 +271,11 @@ int AudioInputALSA::PcmRead(void* buf, uint nbytes)
                     break;
                 case -EINTR:
                 case -EPIPE:
+#if ESTRPIPE != EPIPE
                 case -ESTRPIPE:
                     Recovery(nread);
                     break;
+#endif
```

Comment:
This will cause EPIPE not to be handled correctly. The `#endif` should be moved up so that it only wraps
`case -ESTRPIPE:`